apps/mark/frontend/src/pages/review_page.tsx

```diff
@@ -550,17 +529,13 @@ export function ReviewPage(): JSX.Element {
       {isPortrait ? (
         <ButtonFooter>
           {printMyBallotButton}
-          {languageSettingsButton}
           {settingsButton}
         </ButtonFooter>
       ) : (
         <Sidebar
           footer={
             <React.Fragment>
-              <ButtonFooterLandscape>
-                {languageSettingsButton}
-                {settingsButton}
-              </ButtonFooterLandscape>
+              <ButtonFooterLandscape>{settingsButton}</ButtonFooterLandscape>
```

Comment:
Remind me the story with the `languageSettingsButton`?

Reply:
It's something we'll have to implement eventually, but I noticed it's currently just a placeholder and not rendered, so removed in the meantime to avoid the maintenance cost and just revisit when we're ready for a full implementation